.github/actions/setup-rust/action.yml

```diff
@@ -81,3 +82,7 @@ runs:
       shell: bash
       run: |
         sccache --show-stats
+
+    # Recommended way to install nextest on CI.
+    - name: Install latest nextest release
+      uses: taiki-e/install-action@nextest
```

Comment:
Can we not use `latest` here? I really don't want this to change under our feet at the worst possible time (as things do).

Reply:
very good point. I just copy pasted blindly